lib/dRep/dRepImpl.py

```diff
@@ -48,30 +49,26 @@ class dRep:
     # be found
     def __init__(self, config):
         #BEGIN_CONSTRUCTOR
-        self.callback_url = os.environ['SDK_CALLBACK_URL']
-        self.workspace_url = config['workspace-url']
-        self.srv_wiz_url = config['srv-wiz-url']
-        self.shared_folder = config['scratch']
-        self.config = config
-        self.config['callback_url'] = self.callback_url
-        self.suffix = str(uuid.uuid4())
-
-        self.ws = Workspace(self.workspace_url)
-        self.mgu = MetagenomeUtils(self.callback_url)
-        self.dfu = DataFileUtil(self.callback_url)
-        self.dsu = DataStagingUtils(config)
 
-
-        dprint('os.environ:', os.environ)
-        dprint('config:', config)
+        callback_url = os.environ['SDK_CALLBACK_URL']
+        workspace_url = config['workspace-url']
+        shared_folder = config['scratch']
+        
+        self._globals = { # shared by all API-method runs
```

Comment:
These aren't really global, they are scoped to an instance of the class dRep. You don't have to change this, just an FYI